zenodo/modules/spam/views.py

```diff
@@ -126,17 +127,38 @@ def delete(user_id):
         return render_template('zenodo_spam/delete.html', **ctx)
 
 
-def _evaluate_user_domain(email):
-    email_domain = email.rsplit('@', 1)[-1].lower()
+def normalize_email(email):
+    email = email.lower()
+    username = email[: email.index("@")]
```

Comment:
```suggestion
    username, domain = email.rsplit("@", 1)
```

This makes sure that if someone comes with `alex@slint@gmail.com` we get the domain correctly.

Note that this way you'll get `username == "alex@slint"` and `domain == "gmail.com"` (without the `@` sign), so you'll have to add the `@` later on.